aptos-move/framework/supra-framework/sources/supra_governance.move

```diff
@@ -376,26 +382,26 @@ module supra_framework::supra_governance {
         is_multi_step_proposal: bool,
     ): u64 acquires GovernanceConfig, GovernanceEvents {
         let proposer_address = signer::address_of(proposer);
-        assert!(
-            stake::get_delegated_voter(stake_pool) == proposer_address,
-            error::invalid_argument(ENOT_DELEGATED_VOTER)
-        );
+        // assert!(
```

Comment:
check the proposer are one of the multisig_account owners.

Reply:
Fixed in [0d01950](https://github.com/Entropy-Foundation/aptos-core/pull/36/commits/0d01950612e813fd30c8357b1b4b665c18ce8bb8)